### PR TITLE
Added TPCH

### PR DESCRIPTION
**Pull request by qvad**

### CODE REVIEW

File: sql/tpch/queries/q14.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+SELECT 100.00 * SUM(CASE
+                        WHEN p_type LIKE 'PROMO%' THEN l_extendedprice * (1 - l_discount)
+                        ELSE 0
+    END) / SUM(l_extendedprice * (1 - l_discount)) AS promo_revenue
+FROM lineitem,
+     part
+WHERE l_partkey = p_partkey
+  AND l_shipdate >= DATE '1995-09-01'
+  AND l_shipdate < DATE '1995-09-01';
```
Comment (mtakahar) [inline sql/tpch/queries/q14.sql:9]: ` + interval '1' month` is missing.

Should be:
`AND l_shipdate < DATE '1995-09-01' + INTERVAL '1' month`

File: sql/tpch/queries/q15.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+WITH revenue AS (SELECT l_suppkey                               AS supplier_no,
+                        SUM(l_extendedprice * (1 - l_discount)) AS total_revenue
+                 FROM lineitem
+                 WHERE l_shipdate >= DATE '1996-01-01'
+                   AND l_shipdate < DATE '1996-01-01'
```
Comment (mtakahar) [inline sql/tpch/queries/q15.sql:5]: `+ interval '3' month` is missing.

File: sql/tpch/queries/q20.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+                    FROM partsupp
+                    WHERE ps_partkey IN (SELECT p_partkey
+                                         FROM part
+                                         WHERE p_name LIKE 'forest%')
+                      AND ps_availqty > (SELECT 0.5 * SUM(l_quantity)
+                                         FROM lineitem
+                                         WHERE l_partkey = ps_partkey
+                                           AND l_suppkey = ps_suppkey
+                                           AND l_shipdate >= DATE '1994-01-01'
+                                           AND l_shipdate < DATE '1994-01-01'))
```
Comment (mtakahar) [inline sql/tpch/queries/q20.sql:15]: `+ interval ‘1’ year` is missing.

**Review by mtakahar — CHANGES_REQUESTED** [review]
The INTERVAL clause is missing in some of the queries.

File: sql/tpch/analyze.sql
Comment (mtakahar) [inline sql/tpch/analyze.sql:1]: 7 other tables missing

**Comment by qvad** [comment]
@mtakahar Thank you!
Fixed all comments, currently running tests against latest release and checking results.

**Review by mtakahar — APPROVED** [review]

**Pull request merged**